### PR TITLE
BZ#2065759 Updated vSphere deprecated version requirements

### PR DESCRIPTION
**Pull request by bscott-rh**
Bugzilla link: https://bugzilla.redhat.com/show_bug.cgi?id=2065759
[Doc preview here](https://deploy-preview-43493--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-vsphere-installer-provisioned.html#installation-vsphere-infrastructure_installing-vsphere-installer-provisioned)

### CODE REVIEW

**Comment by bscott-rh** [comment]
@julienlim @racedo Sorry for the tag spam, this is the new PR for the 6.7U3 deprecated note change. [Preview is here](https://deploy-preview-43493--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-vsphere-installer-provisioned.html#installation-vsphere-infrastructure_installing-vsphere-installer-provisioned)

File: modules/installation-vsphere-infrastructure.adoc [hunk 1]
```diff
@@ -59,7 +59,7 @@ You must install the {product-title} cluster on a VMware vSphere version 6 or 7
 
 [IMPORTANT]
 ====
-Installing a cluster on VMware vSphere version 6.7U2 or earlier and virtual hardware version 13 is now deprecated. These versions are still fully supported, but support will be removed in version 4.11 of {product-title}. Hardware version 15 is now the default for vSphere virtual machines in {product-title}. To update the hardware version for your vSphere nodes, see the "Updating hardware on nodes running in vSphere" article.
+Installing a cluster on VMware vSphere version 6.7U3 or earlier and virtual hardware version 13 is now deprecated. These versions are still fully supported, but version 4.11 of {product-title} will require vSphere version 7.0 or later. Hardware version 15 is now the default for vSphere virtual machines in {product-title}. To update the hardware version for your vSphere nodes, see the "Updating hardware on nodes running in vSphere" article.
```
Comment (julienlim) [inline modules/installation-vsphere-infrastructure.adoc:62]: We're still going to support vSphere 6.7U3 in 4.11 in alignment with Kubernetes 1.24.

Change "These versions are still fully supported, but version 4.11 of {product-title} will require vSphere version 7.0 or later." to "These versions are still fully supported, but version 4.11 of {product-title} will require vSphere virtual hardware version 15 or later."

**Comment by julienlim** [comment]
LGTM.  Is the release notes updated in a different PR?

**Comment by lpettyjo** [comment]
@julienlim Yes. I will be updating the rel notes in a separate PR. Stay tuned.

**Comment by ahardin-rh** [comment]
LGTM! I assume this is for 4.10+?

**Comment by bscott-rh** [comment]
> LGTM! I assume this is for 4.10+?

Yes, that's correct

**Pull request merged**